appveyor.yml

```diff
@@ -4,72 +4,69 @@ environment:
     secure: zPYtraT93rgt16J0Egnm0+SD3u8ncUEjAbsquMN5zsY9qowJP/t362GfmtIgbJS4
 
   matrix:
-    - {"nodejs_version": "4", "npm_version": "3"}
-    - {"nodejs_version": "5", "npm_version": "3"}
-    - {"nodejs_version": "6", "npm_version": "3"}
-    - {"nodejs_version": "7", "npm_version": "4"}
-    - {"nodejs_version": "8", "npm_version": "4"}
-    - {"nodejs_version": "9", "npm_version": "5"}
-    - {"nodejs_version": "10", "npm_version": "6"}
-    - {"nodejs_version": "11", "npm_version": "6"}
+  - {"nodejs_version": "6", "npm_version": "3"}
+  - {"nodejs_version": "7", "npm_version": "4"}
+  - {"nodejs_version": "8", "npm_version": "4"}
+  - {"nodejs_version": "9", "npm_version": "5"}
+  - {"nodejs_version": "10", "npm_version": "6"}
+  - {"nodejs_version": "11", "npm_version": "6"}
 
 platform:
-  - x86
-  - x64
+- x86
```

Comment:
Can't get a clue here how to keep node6,7 x64, skipping all other versions for x64...
@royaltm any ideas?

Reply:
https://www.appveyor.com/blog/2018/04/25/specialized-build-matrix-configuration-in-appveyor/